website/client/src/components/userMenu/profilePage.vue

```diff
@@ -24,6 +24,11 @@ export default {
   components: {
     profile,
   },
-  props: ['userId', 'startingPage'],
```

Comment:
the startingPage prop is still used in some places so it cannot be removed, what you can do is having a default value for it https://vuejs.org/v2/guide/components-props.html

Reply:
Before removing 'startingPage' as prop from profilePage.vue (as part of my original changes), I had searched in source code for any references to '<profilePage' which returned 0 results leading to my understanding that profilePage.vue is not used as child component and has no existing parent component. So I removed it as I thought props would be set generally from parent component. Guess I m wrong as you mentioned its being used somewhere else. Pardon my limited knowledge on this as I m a newbie to vue.js, and if I'm missing something obvious.

I have made your requested changes and validated locally and pushed the commit to my fork repo. Please review. Thanks.

Reply:
No worries! The profilePage component is used as <profile-page> in HTML code, that’s why you couldn’t find it.

I’ll give a look at this tomorrow and merge it if everything looks good on Friday.

Thanks!